lib/iris/tests/integration/test_grib2.py

```diff
@@ -49,6 +50,7 @@ def test_gdt1(self):
         cube = load_cube(path)
         self.assertCMLApproxData(cube)
 
+    @skip('iris-grib is currently causing errors')
```

Comment:
I believe this is related (and possibly solved) by [this pull request](https://github.com/SciTools/iris-grib/pull/146).